src/networking.md

```diff
@@ -17,6 +17,12 @@ blockchain](./blockchain-runtime-interface.md#runtime-functions) to connect to m
 to the [mixnet topology](./topology.md). If no external addresses have been published for a
 mixnode, or none of them work, nodes should attempt to discover addresses using the libp2p DHT.
 
+// Here I am not sure, do we have the Dht? or if we are talking about discovery dht, we should name
+// it. Also it implies that we default to the common validator address (an implementor could make all mixnet
+// in a different address (different node than the validating one. Even if not doable with current implementation, this sounds like easy double signing slashing for wrongly configured validators though, also may need a switch in the registration process to register on a single node, may be some smarter conf where we call the transaction to the validator node from the mixnet node to avoid duplicating key, but that is more work)).
+
+// Second thought is this DHT resolution on the discovery risk of spamming validator nodes that purposedly did not run the mixnet, maybe it is a good thing though?
```

Comment:
If a validator is not running the mixnet stuff, it won't ever register a mixnode, and should be completely ignored by mixnet clients.